spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+  describe '#create_factory' do
+    it 'create an array of facilities from a given data' do
+      factory = FacilityFactory.new
+      data = [
```

Comment:
For this part of the project, we wanted to take the data that was coming in from the dmv_data_service and then write a method to take in that data. For example, if we have `co_data = DmvDataService.new.co_dmv_office_locations` and then we do `factory.create_facilities(co_data)`, we should have all the logic in that method to parse through the data that comes in from the dmv_data_service object.